src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java

```diff
@@ -26,11 +35,51 @@
      */

```

Comment:
missing return statement

Reply:
We do not have this agreement for endpoint